VocaDbWeb/Startup.cs

```diff
@@ -116,9 +117,31 @@ public void ConfigureServices(IServiceCollection services)
 				.AddCookie(options =>
 				{
 					options.LoginPath = new PathString("/User/Login");
+					// See https://github.com/aspnet/Security/issues/1569#issuecomment-350947198
+					options.Cookie.SameSite = SameSiteMode.None;

```

Comment:
Set `SameSite` to `SameSiteMode.None`.

Reply:
Could you add a comment why? Is this for external services like UTAU Wiki importer?

Reply:
Yes, it's for external services like UTAU Wiki importer. It's necessary to allow cross-site cookie use (e.g. from `vocaloid.eu` (in which UTAU Wiki importer is located) to `vocadb.net`).